regular-expression.dd

```diff
@@ -13,13 +13,13 @@ $(D_S Regular Expressions,
     Still no amount of fixed functionality could cover all needs, as naturally flexible text data
     needs flexible solutions.
     )
-    $(P Here is where $(LUCKY regular expressions) come in handy, often succinctly called as regexes.
-     Regexes are simple yet powerful language for defining patterns of strings, 
-     put together with a substitution mechanism, they form a Swiss Army knife of text processing.
-     They are considered so useful that a number of languages provides built-in support
-     for regular expressions. This does $(B not) nessary mean that
-     built-in implies $(B faster) processing or more features. It's more a matter of providing
-     $(B convenient and friendly syntax) for typical operations and usage patterns.
+    $(P Here is where $(LUCKY regular expressions) – often just called $(I regexes) – come in handy.
+     Regular expressions are a simple yet comprehensive language for defining string patterns. 
+     Combined with a mechanism for string substitution, regular expressions are a powerful tool for text processing.
+     In fact, they are considered so useful that a number of languages provide built-in support
+     for them. However, built-in support does $(I not) necessarily imply $(I faster) processing 
```

Comment:
I don't think the italics on this line + lines 22 add anything to the doc. IMO, the emphasis the italics provide makes the sentence read a bit oddly.